test_view_model.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from view_model import ViewModel
+from datetime import datetime, timedelta
+from todo_item import TodoItem
+import os
+
+
+@pytest.fixture
+def view_model():
+    _DEFAULT_ITEMS = [
+        {
+            "idShort": 1,
+            "idList": os.getenv("LIST_ID_NOT_STARTED"),
```

Comment:
just to check, I thought things like 'LIST_ID_NOT STARTED' was supposed to be private? that's why it is in the .env file which isn't committed to git. 

if I set them within this file surely they will then become visible when committed to git?

Reply:
Since this is a unit test that doesn't talk to Trello it doesn't need to use real values, it should work as long as the three list ids are different, so you could do e.g.
```python
os.environ['LIST_ID_NOT_STARTED'] = 'not-started-list-id'
os.environ['LIST_ID_IN_PROGRESS'] = 'in-progress-list-id'
os.environ['LIST_ID_DONE'] = 'done-list-id'
```

Reply:
instead of this i have loaded the the .env.test file into test_view_model.py like i did in test_app.py.